crates/typst/src/visualize/color.rs

```diff
@@ -1840,6 +1840,11 @@ impl ColorSpace {
             _ => None,
         }
     }
+
+    /// Returns whether this color space is part of the Oklab family.
+    pub fn is_oklab_family(&self) -> bool {
```

Comment:
is this used?

Reply:
Nope, it was used to add more intermediary stops, but 32 seems to be enough in most cases.